src/shared/components/emptyList/emptyList.css

```diff
@@ -0,0 +1,14 @@
+.emptyCart-container{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+.empty-cart-image{
+    width:80vh;
+    height:400px;
```

Comment:
Suggestion: Can give height in rems

Reply:
I want this height to be static . I don't want to keep it variable.